manifests/g/GitHub/GitHubDesktop/Beta/3.4.2-beta3/GitHub.GitHubDesktop.Beta.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GitHub.GitHubDesktop.Beta
+PackageVersion: 3.4.2-beta3
+MinimumOSVersion: 10.0.0.0
+Platform:
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-06-12
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://desktop.githubusercontent.com/github-desktop/releases/3.4.2-beta3-6edaffed/GitHubDesktopSetup-x64.exe
+  InstallerSha256: 9C0CD261BEDBD8E570E01354E568F33A5F194F5AE67E4CB8F4DAE85671C90C36
+  InstallerSwitches:
+    Silent: /s
+    SilentWithProgress: /s
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://desktop.githubusercontent.com/github-desktop/releases/3.4.2-beta3-6edaffed/GitHubDesktopSetup-x64.msi
+  InstallerSha256: 0FAF2FD887F72A441E7F9B912A3C681ACCED8FE67E0CED8DF2D43CD12C5C8BEE
+  ProductCode: '{8ABF5209-DD0D-497F-91AF-4C6293B3222C}'
+  AppsAndFeaturesEntries:
+  - DisplayName: GitHub Desktop Deployment Tool
+    DisplayVersion: 3.4.2.0
+    UpgradeCode: '{00D8E2EE-13EA-5BEB-87F0-70EFC46A7D4A}'
```

Comment:
```suggestion
```

Should be removed according to https://github.com/microsoft/winget-pkgs/pull/159350#issuecomment-2186908992